tornado-api/src/main/java/uk/ac/manchester/tornado/api/profiler/JSonWriter.java

```diff
@@ -0,0 +1,258 @@
+package uk.ac.manchester.tornado.api.profiler;
```

Comment:
Apply license

Reply:
Hmm code formatter should do this. But yes.  Agree 100%